spswarehouse/powerschool/calpads.py

```diff
@@ -0,0 +1,220 @@
+import time
+from spswarehouse.powerschool.powerschool import (
```

Comment:
Yeah, this import reinforces that PowerSchool should be a class, and then this module would be a derived class from the PowerSchool class.